docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,108 @@
+# Solidity Mutation Verifier Based on Gambit
+
+This is a mutation verifier which
+ checks that variants of the original
+ solidity program do not pass the specification.
+If a mutated program passes the specification,
+it may indicate that the specification is vacuous or not rigorous enough.
+It uses mutations from the [Gambit](https://github.com/Certora/gambit)
+  mutation generator.
+In the rest of the document,
+  we refer to both the mutation generator and the verifier as Gambit.
+
+## Running the Mutation Verifier
+
+- Example:
+```
+java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/Example.conf
+```
+
+- Remember to `yarn install` or `npm install` when working
+  on customer code if required by the customer code infra!
+
+- **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
+
+- Gambit **does** support `--staging`!
```

Comment:
Link to the command-line options in the Prover manual using, e.g.
```
{ref}`--staging`
```

You may need to add a ref to `/docs/prover/cli/options.md`, search for `--rule_sanity` in that document for an example

Reply:
hmm if you can help with this, that would be great. Not sure I am following what you are saying.